src/pspm_con1.m

```diff
@@ -130,26 +130,27 @@ function pspm_con1(modelfile, connames, convec, datatype, deletecon, options)
   % there are issues if data.stats has NaN and the corresponding conmat
   % also contains 0
   idx_issue = any(isnan(data.stats),2);
-  l_conmat_r = length(conmat(:,1)); 
+  l_conmat_r = length(conmat(:,1));
+  data_stats_converted = data.stats;
+  conmat_converted = conmat;
   if any(idx_issue(:))
     for i_conmat_r = 1:l_conmat_r % if conmat is a 2D matrix, repeat for each row
       conmat_array = conmat(i_conmat_r,:);
       idx_valid_issue = transpose(conmat_array==0) .* idx_issue;
       idx_invalid_issue = transpose(conmat_array~=0) .* idx_issue;
-      if any(idx_valid_issue) && ~any(idx_invalid_issue)
-        warning(['Calculated data.stats contain NaNs that are caused by unknown reasons. '...
-          'data.stats are then used in a contrasts, producing an invalid results (NaN).']);
-        data_stats_converted = data.stats;
-        data_stats_converted(isnan(data_stats_converted)) = 0;
-        conval = conmat * data_stats_converted;
-      else
+      if any(idx_valid_issue)
```

Comment:
We should be careful here, the NaN are not a problem if and only if there are no `invalid issue`.
Shouldn't the text be `if any(idx_invalid_issue)` ?